app/views/posts/_post.html.erb

```diff
@@ -38,7 +38,7 @@
       <span class='text-danger'>(deleted <%= distance_of_time_in_words(post.created_at, post.deleted_at, include_seconds: true) %> after creation)</span>
     <% end %>
 
-    <% if post.comments.any? %>
+    <% if post.comments.any? and not hide_feedbacks %>
```

Comment:
Typically in ruby we prefer `&&` for `and` and `!` for `not`, so this really should be `&& !hide_feedbacks` (in terms of code style). Not sure why we want to hide comments if feedbacks are hidden though (or what the `hide_feedbacks` method/variable is).